docs/csharp/language-reference/operators/user-defined-conversion-operators.md

```diff
@@ -15,7 +15,7 @@ helpviewer_keywords:
 
 A user-defined type can define a custom implicit or explicit conversion from or to another type.
 
-Implicit conversions don't require special syntax to be invoked and can occur in a variety of situations, for example, in assignments and methods invocations. Predefined C# implicit conversions always succeed and never throw an exception or lose information. User-defined implicit conversions should behave in that way as well. If a custom conversion can throw an exception or lose information, define it as an explicit conversion.
```

Comment:
I've removed the "or lose information" words because some implicit numeric conversions (for example, `ulong` to `float`) lose precision.

Reply:
great catch.